src/events/aws_sdk.js

```diff
@@ -546,6 +546,24 @@ const stepFunctionsEventCreator = {
 
             break;
         }
+        case 'sendTaskSuccess': {
+            let output;
```

Comment:
This code is a duplication of handling `startExecution`

Reply:
fixed